.github/workflows/test.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Test source code"
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+  RunGoTests:
+    # I use MacOS, becuse it doesn't need additional libs (in contrast to linux).
```

Comment:
I don't see it as an advantage... This means hidden dependencies and might break in the future. Add a Linux build as well, please.

Reply:
I see it like this.
1. It's free for open source projects
2. You need to test locally anyway. If your tests/build pass locally, it's 99% likely to pass on CI, so you don't need to wait
3. The maintainer/reviewer won't look at your PR *exactly* as you push commits.

So it doesn't really matter that it takes some time. If it took an hour or so, *that* would be a problem.

Reply:
> It's free for open source projects

oh excellent! I tought it was limited to some amount of mintes even for public repos, but it actually isn't :tada:

![image](https://user-images.githubusercontent.com/73652197/207087126-13887d99-0645-4801-a788-8bd1708dff8f.png)

So, no more opjections!